core/templates/dev/head/base.html

```diff
@@ -56,7 +56,9 @@
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
     <title itemprop="name">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
-
+    {% if inSearchPage %}
```

Comment:
This just feels a bit odd, the way it's written. Instead, can you do this as {% block base_url %}{% endblock base_url %} and overwrite it in the search page?